Lib/test/list_tests.py

```diff
@@ -61,7 +61,7 @@ def test_repr(self):
 
     def test_repr_deep(self):
         a = self.type2test([])
-        for i in range(sys.getrecursionlimit() + 100):
+        for i in range(3_000):
```

Comment:
Shouldn't this use C_RECURSION_LIMIT like the other tests? (also mapping_tests.py, test_dict.py, test_exception_group.py)